packages/transition-common/src/services/jobs/Job.ts

```diff
@@ -69,11 +72,13 @@ export interface JobAttributes<TData extends JobDataType> {
     status: JobStatus;
     statusMessages?: { errors?: ErrorMessage[]; warnings?: ErrorMessage[]; infos?: ErrorMessage[] };
     /**
-     * Data internal to the job management, that is not relevant to the users
+     * Data internal to the job management, that is not relevant to the users.
+     * Typically volatile during job execution and can be used for example to
```

Comment:
Not sure the word volatile is the word you want to use here. A checkpoint is definitely something that is not volatile